### PR TITLE
Fixes Yaml Powershell tasks for bicep deployment with variables with '$'

### PR DESCRIPTION
**Pull request by Soruk**
This PR fixes the issue #210

### CODE REVIEW

**Comment by Soruk** [comment]
I agree with the CLA and I have sole ownership of intellectual property rights to my Submissions and I am not making Submissions in the course of work for my employer.

**Comment by jamesmontemagno** [comment]
Thank you so much @Soruk  this is awesome and makes a lot of sense!

File: .github/workflows/template-web.yml [hunk 1]
```diff
@@ -70,7 +70,20 @@ jobs:
...
+            $servicePlanSku = '${{ secrets.SERVICE_PLAN_SKU }}'
+            $hubWebAppName = '${{ secrets.HUB_WEBAPP_NAME }}'
+            $apiName = '${{ secrets.API_RESOURCE_NAME }}'
+           
+            az deployment group create -n "$($deploymentName)" `
+               --resource-group  "$($resourceGroupName)" `
+               --template-file deploy/Web/web.bicep `
+               --parameters webAppName="$($webAppName)" servicePlanName="$($servicePlanName)" servicePlanSku="$($servicePlanSku)" hubWebAppName="$($hubWebAppName)" apiName="$($apiName)" 
+               
+          #  az deployment group create -n ghaction -g ${{ secrets.AZURE_RESOURCE_GROUP_NAME }} --template-file deploy/Web/web.bicep --parameters webAppName=${{ secrets.WEBAPP_NAME }} servicePlanName=${{ secrets.SERVICE_PLAN_NAME }} servicePlanSku=${{ secrets.SERVICE_PLAN_SKU }} hubWebAppName=${{ secrets.HUB_WEBAPP_NAME }} apiName=${{secrets.API_RESOURCE_NAME}} updaterName=${{secrets.UPDATER_RESOURCE_NAME}}
```
Comment (MarcusFelling) [inline .github/workflows/template-web.yml:86]: can we remove this commented out script?

Comment (Soruk) [reply .github/workflows/template-web.yml:86]: Yes, I forgot to do it.
Wait a minute

Comment (Soruk) [reply .github/workflows/template-web.yml]: @MarcusFelling  done

Comment (MarcusFelling) [reply .github/workflows/template-web.yml]: LGTM! Thanks, @Soruk!

**Review by MarcusFelling — APPROVED** [review]

**Pull request merged**